install_config/install/advanced_install.adoc

```diff
@@ -959,6 +959,9 @@ openshift_master_cluster_method=native
 openshift_master_cluster_hostname=openshift-cluster.example.com
 openshift_master_cluster_public_hostname=openshift-cluster.example.com
 
+# apply updated node defaults
+openshift_node_kubelet_args={'pods-per-core': ['10'], 'max-pods': ['250'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}
```

Comment:
We should check these into the product too, if you think they are the correct numbers.